xml/Microsoft.Extensions.DependencyInjection/ObjectFactory.xml

```diff
@@ -5,7 +5,7 @@
   <TypeSignature Language="VB.NET" Value="Public Delegate Function ObjectFactory(serviceProvider As IServiceProvider, arguments As Object()) As Object " />
   <TypeSignature Language="C++ CLI" Value="public delegate System::Object ^ ObjectFactory(IServiceProvider ^ serviceProvider, cli::array &lt;System::Object ^&gt; ^ arguments);" />
   <TypeSignature Language="F#" Value="type ObjectFactory = delegate of IServiceProvider * obj[] -&gt; obj" />
-  <TypeSignature Language="C#" Value="public delegate object ObjectFactory(IServiceProvider serviceProvider, object[]? arguments);" FrameworkAlternate="dotnet-plat-ext-5.0" />
+  <TypeSignature Language="C#" Value="public delegate object ObjectFactory(IServiceProvider serviceProvider, object?[]? arguments);" FrameworkAlternate="dotnet-plat-ext-5.0" />
```

Comment:
Hmm, this doesn't look quite right.

Reply:
Well it matches the source code so I guess this is expected. https://github.com/dotnet/runtime/blob/f6d8e88bb2f41ad203925d67e87e01603810f7c2/src/libraries/Microsoft.Extensions.DependencyInjection.Abstractions/src/ObjectFactory.cs#L14